js/stores/appStore.js

```diff
@@ -239,7 +239,7 @@ function windowDefaults () {
     windowOffset: 20,
     webPreferences: {
       sharedWorker: true,
-      partition: ''
+      partition: 'main-1'
```

Comment:
> Private tabs should use the in-memory '' partition and the browser process should load favicons through the partition that matches the one used by the tab.

The same logic would apply to search autocomplete queries. They aren't strictly associated with a tab, but I guess they should go through the partition of the active tab when the query is entered.